### PR TITLE
Pull request for issues #397 and #457 and #469

### PR DESCRIPTION
**Pull request by Iridar**
#397 - Add a config var to allow changing default weapon tint for randomly generated soldiers.
#457 - Add an Event to X2TacticalGameRuleset that will trigger after Aliens are spawned.

### CODE REVIEW

File: X2WOTCCommunityHighlander/Src/XComGame/Classes/XGCharacterGenerator.uc [hunk 1]
```diff
@@ -840,9 +843,9 @@ function SetArmorTints(X2CharacterTemplate CharacterTemplate)
 		}
 	}
 
-	//For generated soldiers, weapon tint now defaults to no tint
-	kSoldier.kAppearance.iWeaponTint = 20;
-
+	// Begin issue #397
+	kSoldier.kAppearance.iWeaponTint = iDefaultWeaponTint;
```
Comment (Xymanek) [inline X2WOTCCommunityHighlander/Src/XComGame/Classes/XGCharacterGenerator.uc]: Why not an event? Maybe someone will want to make a mod that randomizes the colour for each soldier

Comment (Iridar) [reply X2WOTCCommunityHighlander/Src/XComGame/Classes/XGCharacterGenerator.uc]: The possibility didn't occur to me, wouldn't really know how to set it up anyway. Using config is enough for me.

Comment (MrNiceUK) [reply X2WOTCCommunityHighlander/Src/XComGame/Classes/XGCharacterGenerator.uc]: To be fair, there is no `NewGameState` accessible to trigger the event with! Indeed such a game state may not even exist, depending when and where the generator is being used.

File: CHANGELOG.md [hunk 2]
```diff
@@ -217,7 +219,7 @@ All notable changes to Vanilla 'War Of The Chosen' Behaviour will be documented
   no broken duo poses for units that can't play them (#309)
 - Additional Photobooth particle system enums for mods (#359)
 - Tweaks to "Resistance Archives" random Legacy Operations UI. Restarts show the correct locked difficulty, and a crash condition on backing out of a restart fixed. (#307)
-
+- Added 5 new inventory slots. (#469)
```
Comment (robojumper) [inline CHANGELOG.md]: We don't need to mention this in the changelog; at least we haven't before.

Comment (Iridar) [reply CHANGELOG.md]: Do you want me to cancel the change then?

Comment (robojumper) [reply CHANGELOG.md]: Just remove it from the changelog, yeah

Comment (Iridar) [reply CHANGELOG.md]: Roger.

**Comment by Iridar** [comment]
Thank you, appreciate your patience.

**Pull request merged**